modules/op-about-taskrun.adoc

```diff
@@ -0,0 +1,38 @@
+// Ths module is included in the following assembly:
+//
+// *openshift_pipelines/op-creating-applications-with-cicd-pipelines.adoc
+
+[id="about-taskrun_{context}"]
+= About TaskRun
+
+A _TaskRun_ instantiates a Task for execution with specific inputs, outputs, and execution parameters on a cluster. It can be invoked on its own or as part of a Pipeline.
+
+A Task consists of one or more Steps that execute container images, and each container image performs a specific piece of build work. A TaskRun executes the Steps in a Task in the specified order, till all Steps execute successfully or a failure occurs.
+
+Here is an example of a TaskRun to run a Pipeline `build-and-deploy` with relevant input and output parameters:
```

Comment:
This looks like a copy&paste... this run a Task, not a Pipeline called `apply-manifests` with relevant input (and param), there are no outputs